screen-capture-api/region-capture/index.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Region Capture API demo</title>
+  <link href="index.css" rel="stylesheet">
+  <script defer src="index.js"></script>
+</head>
+
+<body>
+  <h1>Region Capture API example</h1>
+  <p>
+    This example captures the demo region of the screen.
+    Click the Start Capture button to begin.
+  </p>
```

Comment:
Same comment here, it's hard for me to know if it's working or not compared to the other ones